src/str.c

```diff
@@ -1381,6 +1597,8 @@ int STk_init_string(void)
   ADD_PRIMITIVE(strgei);
   ADD_PRIMITIVE(substring);
   ADD_PRIMITIVE(string_append);
+  ADD_PRIMITIVE(string_append_adjsize);
```

Comment:
Could you use please dstring_append (for destructive string-append)? This is how function are named in C when they modify their first parameter and a non destructive version of the function exists.

Reply:
Sure! I'll work on this later (maybe tomorrow in the morning)